prepare-cluster/action.yml

```
@@ -88,6 +88,14 @@ inputs:
   kots-wait-duration:
     description: 'Timeout for KOTS to be used while waiting for individual components to be ready. must be in Go duration format (eg: 10s, 2m) (default "2m")'
     required: false
+  customer-entitlements:
+    description: |
+      Entitlements to assign to the customer.
+      Example:
+      <pre>customer-entitlements: \|
+        - name: "number-of-users"
+          value: "10"</pre>.'
```

i think there are some extra characters here

```suggestion
          value: "10"</pre>
```